exchange/bitswap/workers.go

```diff
@@ -182,10 +182,11 @@ func (bs *Bitswap) rebroadcastWorker(parent context.Context) {
 	defer cancel()
 
 	broadcastSignal := time.After(rebroadcastDelay.Get())
+	tick := time.Tick(10 * time.Second)
 
 	for {
 		select {
-		case <-time.Tick(10 * time.Second):
```

Comment:
interesting. should grep the codebase and make sure this doesn't happen elsewhere. Was it just not getting gc-ed? or is there some releasing we need to do?